downloadDirectory/src/test/java/downloadDirectory/TestProtegeExporter.java

```diff
@@ -52,16 +64,51 @@ public void testProtegeExporterIT() throws SQLException, IOException
 		ProtegeExporter testExporter = new ProtegeExporter();
 		
 		testExporter.setReleaseDirectory("/home/sshorser/workspaces/reactome/Release");
-		testExporter.setPathToWrapperScript(PATH_TO_JAVA_ROOT + "src/main/resources/");
+		testExporter.setPathToWrapperScript(pathToJavaRoot + "src/main/resources/");
 		testExporter.setExtraIncludes(Arrays.asList("-I/home/ubuntu/perl5/lib/perl5/","-I/home/sshorser/perl5/lib/perl5/"));
 		testExporter.setParallelism(4);
-		testExporter.setPathwayIdsToProcess(new HashSet<>(Arrays.asList(1670466L,8963743L,870392L,1500931L,5205647L)));
+		HashSet<Long> pathwayIds = new HashSet<>(Arrays.asList(1670466L,8963743L,870392L,1500931L,5205647L));
```

Comment:
Could this be declared as Set<Long>?

Reply:
Yes, but automatic refactoring chose to be specific and declare it a `HashSet`.

Reply:
Okay, I see.  It is better practice to declare the interface and assign the implementation, though.

Reply:
Yes, I know that, it will be changed.

Reply:
Okay, thanks!